git-cliff-core/src/changelog.rs

```diff
@@ -479,37 +495,44 @@ impl<'a> Changelog<'a> {
 	}
 
 	/// Adds remote data (e.g. GitHub commits) to the releases.
-	pub fn add_remote_data(&mut self) -> Result<()> {
+	pub fn add_remote_data(&mut self, range: Option<&str>) -> Result<()> {
 		debug!("Adding remote data...");
 		self.add_remote_context()?;
 
+		let range_head = range.and_then(|r| r.split("..").last());
+		// Create variable that is None if head is "HEAD", otherwise head
+		let head = match range_head {
+			Some("HEAD") => None,
+			other => other,
+		};
```

Comment:
```suggestion
		// Create variable that is None if head is "HEAD", otherwise head
		let range_head = range.and_then(|r| r.split("..").last());
		let head = match range_head {
			Some("HEAD") => None,
			other => other,
		};
```